backend/tessend/tessend.go

```diff
@@ -553,7 +566,7 @@ func (t *Tessend) getPerResourceMetrics(now int64, data *Data) {
 		count, err := etcd.Count(t.ctx, t.client, metricFunc(t.ctx, ""))
 		if err != nil {
 			logger.WithError(err).Error("unable to retrieve resource count")
-			continue
+			return
```

Comment:
Are we returning here because in the case of an error on `etcd.Count`, the backend will be restarted anyway?

Reply:
Yes.